dist/README.md

```diff
@@ -1,10 +1,54 @@
-## Running ovn-kubernetes with ovsdb-etcd
-
-In order to run [ovn-kubernetes](https://github.com/ovn-org/ovn-kubernetes) with ovsdb-etcd:
-* set the environment variable `OVSDB_ETCD_REPOSITORY` with your image repository name
-* optionally set `OVN_KUBERNETES_ROOT` to location of ovn-kubernetes repository, 
-  default is <this repository>/./../../ovn-org/ovn-kubernetes )  
-* run `make ovn-kubernetes-build` 
-* run kind.sh to deploy ovn-kubernetes 
-
-The ovn-kubernetes files are from commit e567a39c76930a26bdf1f312c36fa716f18df9a7
\ No newline at end of file
+# Deplyment of ovsdb-etcd on OVN kubernetes
+In order to deploy [OVN kubernetes](https://github.com/ovn-org/ovn-kubernetes) with ovsdb-etcd follow the following steps:
+
+1. prepare OVN kubernetes (for more information read [OVN kubernetes KIND deployment official documentation](https://github.com/ovn-org/ovn-kubernetes/blob/master/docs/kind.md))
+	- clone OVN kubernetes from the main repository.
+		```bash
+		# Clone ovn-kubernetes repo
+		git clone https://github.com/ovn-org/ovn-kubernetes
+		cd ovn-kubernetes
+		```
+	- build the OVN kubernetes container image.
+		```bash
+		pushd go-controller
+		make
+		popd
+		pushd dist/images
+		make fedora
+		popd
+		```
+1. prepare ovsdb-etcd
+	- clone `ovsdb-etcd` from the main repository.
+		```bash
+		# Clone ovsdb-etcd repo
+		git clone https://github.com/IBM/ovsdb-etcd
+		cd ovsdb-etcd
+		```
+	- login to your account of common image repository e.g. docker hub or quay.
+	- set the environment variable `OVSDB_ETCD_REPOSITORY` with your image repository name. for example:
+		```bash
+		export OVSDB_ETCD_REPOSITORY=omerbh
+		```
+	- set the environment variable `OVN_KUBERNETES_ROOT` to the location of OVN kubernetes repository. for example:
+		```bash
+		export OVN_KUBERNETES_ROOT=$HOME/go/src/github.com/ovn-org/ovn-kubernetes
+		```
+	    (default value : `$HOME/work/src/github.com/ovn-org/ovn-kubernetes`)
```

Comment:
please return the original default value

Reply:
Done, Thanks.